securedrop-admin

```diff
@@ -0,0 +1,159 @@
+#!/usr/bin/env python2
+"""
+SecureDrop Admin Toolkit.
+
+For use by administrators to install, maintain, and manage their SD
+instances.
+"""
+
+import argparse
+import subprocess
+import os
+import sys
+import getpass
+import logging
+
+sdlog = logging.getLogger('SD')
```

Comment:
It's true that is generally good practice to keep module imports free from side effects; however, it is alright to make an exception for the `logging` module because it was _specifically designed_ to be used in this manner. See the [Advanced Logging Tutorial](https://docs.python.org/2/howto/logging.html#advanced-logging-tutorial)).

Note that if @msheiny followed this advice and instantiated `sdlog` in `if __name__ == '__main__'`, he would have to make further changes to make `sdlog` available to every function that wishes to log: he could pass it by argument to every function that wishes to log, or use a global, or refactor into an object that handles logging—all of which are either bad practice or are ridiculously over-engineered for what should be a simple and straightforward script.